app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en" class="govuk-template">
   <head>
-    <%= render 'layouts/google_tag_manager' if google_tag_present? %>
+    <%= render partial: 'layouts/google_tag_manager' if google_tag_present? %>
```

Comment:
I'm a little surprised that has any effect, but as it will do no harm ....

Reply:
Just to clarify, the default behaviour of `render` is to render a partial. As stated in the [docs](https://api.rubyonrails.org/classes/ActionController/Renderer.html#method-i-render): 

> If no options hash is passed or if :update is specified, the default is to render a partial and use the second parameter as the locals hash.

Reply:
Interesting. Thanks.